tools/release_do_nothing.py

```diff
@@ -0,0 +1,758 @@
+# Copyright Iris contributors
+#
```

Comment:
File wide:
- Perhaps making section titles clearer, and numbering steps will help avoid accidentally skipping a step?
- "DONE" is a bit of a nuisance to type each time, perhaps change to "Y"?

Reply:
e23eb9009